content/blog/2021-04-30-april-21-community-gems.md

```diff
@@ -0,0 +1,183 @@
+---
+title: April '21 Community Gems
+date: 2021-04-30
+description: |
+  A roundup of technical Q&A's from the DVC community. 
+  This month: remote storage integration, removing old experiments,
+  ideas for running CML pipeline reports and more.
+descriptionLong: |
+  A roundup of technical Q&A's from the DVC community. 
```

Comment:
we should start utilizing these longer descriptions somehow. E.g. make `DVC and CML` and link them to the anchors on the page. Overall this field support basic Markdown, so I would think how can we make it look nice and fun.